website/content/api-docs/system/internal-ui-version.mdx

```diff
@@ -0,0 +1,49 @@
+---
+layout: api
+page_title: /sys/internal/ui/version - HTTP API
+description: >-
+  The `/sys/internal/ui/version` endpoint exposes the software version of Vault.
+---
+
+# `/sys/internal/ui/version`
+
+The `/sys/internal/ui/version` endpoint is used to expose the software version
+of Vault to the UI so that it can be displayed to logged in users.
+
+This is currently only being used internally for the UI. Due to the nature of its
+intended usage, there is no guarantee on backwards compatibility for this endpoint.
```

Comment:
```suggestion
Vault uses internal endpoints to provide information to the Vault UI and
CLI. Internal endpoints are explicitly intended to support Vault
functionality, so we do not recommend them for general use
and do not guarantee backwards compatibility across versions.
```

Style correction: avoid "this" as a pronoun. Make it explicit that we recommend against general use and it's a "use at your own risk" situation

Reply:
I don't think that there's any practical use by the CLI of the internal endpoints. Are you ok if I take your suggestion minus the `and CLI` (lines 13 and 14).